src/sentry/services/hybrid_cloud/integration/impl.py

```diff
@@ -231,10 +231,7 @@ def get_organization_contexts(
             integration_id=integration.id,
             organization_id=organization_id,
         )
-        return (
-            serialize_integration(integration),
-            organization_integrations,
-        )
```

Comment:
Correct. We were getting away with it only because `RpcIntegration` is duck-typedly similar enough to `Integration` to pass to the serialization function without consequence (but that might not always be true).